### ISSUE TITLE
Downstream test failures

### ISSUE THREAD

**Issue by WhyNotHugo** [python-lsp/python-lsp-black#58]
Some tests are failing on Alpine Edge. These were passing when py3-lsp-black was upgraded to 2.0.0, so it must be due to some recent dependency update:

```
___________________________________________ test_load_config_defaults ____________________________________________

config = {'fast': False, 'line_length': 88, 'preview': False, 'pyi': False, ...}

    def test_load_config_defaults(config):
        config = load_config(str(fixtures_dir / "example.py"), config)
    
>       assert config == {
            "line_length": 88,
            "target_version": set(),
            "pyi": False,
            "fast": False,
            "skip_magic_trailing_comma": False,
            "skip_string_normalization": False,
            "preview": False,
        }
E       AssertionError: assert {'line_length': 88, 'fast': False, 'pyi': False, 'skip_string_normalization': False, 'skip_magic_trailing_comma': False, 'target_version': {<TargetVersion.PY38: 8>, <TargetVersion.PY311: 11>, <TargetVersion.PY310: 10>, <TargetVersion.PY39: 9>}, 'preview': False} == {'line_length': 88, 'target_version': set(), 'pyi': False, 'fast': False, 'skip_magic_trailing_comma': False, 'skip_string_normalization': False, 'preview': False}
E         
E         Common items:
E         {'fast': False,
E          'line_length': 88,
E          'preview': False,
E          'pyi': False,
E          'skip_magic_trailing_comma': False,
E          'skip_string_normalization': False}
E         Differing items:
E         {'target_version': {<TargetVersion.PY38: 8>, <TargetVersion.PY311: 11>, <TargetVersion.PY310: 10>, <TargetVersion.PY39: 9>}} != {'target_version': set()}
E         
E         Full diff:
E           {
E               'fast': False,
E               'line_length': 88,
E               'preview': False,
E               'pyi': False,
E               'skip_magic_trailing_comma': False,
E               'skip_string_normalization': False,
E         -     'target_version': set(),
E         ?                       ^^^^^^
E         +     'target_version': {
E         ?                       ^
E         +         <TargetVersion.PY38: 8>,
E         +         <TargetVersion.PY39: 9>,
E         +         <TargetVersion.PY311: 11>,
E         +         <TargetVersion.PY310: 10>,
E         +     },
E           }

tests/test_plugin.py:282: AssertionError
_______________________________________ test_load_config_with_skip_options _______________________________________

config_with_skip_options = <pylsp.config.config.Config object at 0x7fb001c24990>

    def test_load_config_with_skip_options(config_with_skip_options):
        config = load_config(
            str(fixtures_dir / "skip_options" / "example.py"), config_with_skip_options
        )
    
>       assert config == {
            "line_length": 88,
            "target_version": set(),
            "pyi": False,
            "fast": False,
            "skip_magic_trailing_comma": True,
            "skip_string_normalization": True,
            "preview": False,
        }
E       AssertionError: assert {'line_length': 88, 'fast': False, 'pyi': False, 'skip_string_normalization': True, 'skip_magic_trailing_comma': True, 'target_version': {<TargetVersion.PY38: 8>, <TargetVersion.PY311: 11>, <TargetVersion.PY310: 10>, <TargetVersion.PY39: 9>}, 'preview': False} == {'line_length': 88, 'target_version': set(), 'pyi': False, 'fast': False, 'skip_magic_trailing_comma': True, 'skip_string_normalization': True, 'preview': False}
E         
E         Common items:
E         {'fast': False,
E          'line_length': 88,
E          'preview': False,
E          'pyi': False,
E          'skip_magic_trailing_comma': True,
E          'skip_string_normalization': True}
E         Differing items:
E         {'target_version': {<TargetVersion.PY38: 8>, <TargetVersion.PY311: 11>, <TargetVersion.PY310: 10>, <TargetVersion.PY39: 9>}} != {'target_version': set()}
E         
E         Full diff:
E           {
E               'fast': False,
E               'line_length': 88,
E               'preview': False,
E               'pyi': False,
E               'skip_magic_trailing_comma': True,
E               'skip_string_normalization': True,
E         -     'target_version': set(),
E         ?                       ^^^^^^
E         +     'target_version': {
E         ?                       ^
E         +         <TargetVersion.PY38: 8>,
E         +         <TargetVersion.PY39: 9>,
E         +         <TargetVersion.PY311: 11>,
E         +         <TargetVersion.PY310: 10>,
E         +     },
E           }

tests/test_plugin.py:298: AssertionError
================================================ warnings summary ================================================
tests/test_plugin.py:8
  /home/hugo/src/gitlab.alpinelinux.org/alpine/aports/testing/py3-lsp-black/src/python-lsp-black-2.0.0/tests/test_plugin.py:8: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
    import pkg_resources

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================ short test summary info =============================================
FAILED tests/test_plugin.py::test_load_config_defaults - AssertionError: assert {'line_length': 88, 'fast': False, 'pyi': False, 'skip_string_normalization': False, '...
FAILED tests/test_plugin.py::test_load_config_with_skip_options - AssertionError: assert {'line_length': 88, 'fast': False, 'pyi': False, 'skip_string_normalization': True, 's...
============================= 2 failed, 17 passed, 2 deselected, 1 warning in 0.25s ==============================
```

Apparently, the tests in https://github.com/python-lsp/python-lsp-black/issues/57 are also failing too.

**Comment by WhyNotHugo** [comment]
Exact package versions:

```
black==24.3.0
click==8.1.7
docstring-to-markdown==0.15
iniconfig==2.0.0
installer==0.7.0
jedi==0.19.1
mypy-extensions==1.0.0
packaging==24.0
parso==0.8.3
pathspec==0.12.1
platformdirs==4.2.0
pluggy==1.4.0
py==1.11.0
pyparsing==3.1.2
pytest==0.0.0
python-lsp-jsonrpc==1.1.2
python-lsp-server==1.10.1
typed-ast==1.5.5
ujson==5.9.0
```

**Comment by ccordoba12** [comment]
Hey @WhyNotHugo, thanks for reporting. This was already fixed in PR #56.

**Comment by WhyNotHugo** [comment]
I missed that, thanks
